### ISSUE TITLE
Any finetuning is having No effect on small models 

### ISSUE THREAD

**Issue by VatsaDev** [Lightning-AI/litgpt#627]
Hi, I've tried finetuning Pythia 410m deduped using adapter, adapter_v2, and full finetune, for the Alpaca and Dolly datasets. However, the model appears to never learn the format of any of those datasets, and at every step/iter the loss is between 0.4 and 6 varies randomly. 

Heres the colab notebook for finetuning -> https://colab.research.google.com/drive/13NHAZk4TDYJsGh143BuhIQRIrm6ndCcx?usp=sharing

Am I doing something wrong or is there a bug?

**Comment by VatsaDev** [comment]
@carmocca Any thoughts on this?

**Comment by carmocca** [comment]
You'll need to tweak hyperparameters. Sorry, I haven't trained these models myself.
